hack/tools/install.sh

```diff
@@ -19,17 +19,17 @@ set -euo pipefail
 readonly PKG_ROOT="$(git rev-parse --show-toplevel)"
 
 # https://github.com/aws/aws-cli/tags
-AWSCLI_VERSION="2.18.5"
+AWSCLI_VERSION="2.19.4"
 # https://github.com/helm/chart-testing
 CT_VERSION="v3.11.0"
 # https://github.com/eksctl-io/eksctl
-EKSCTL_VERSION="v0.192.0"
+EKSCTL_VERSION="v0.194.0"
 # https://github.com/onsi/ginkgo
-GINKGO_VERSION="v2.20.2"
+GINKGO_VERSION="v2.21.0"
 # https://github.com/golangci/golangci-lint
-GOLANGCI_LINT_VERSION="v1.61.0"
+GOLANGCI_LINT_VERSION="v1.62.0"
 # https://github.com/hairyhenderson/gomplate
-GOMPLATE_VERSION="v4.1.0"
+GOMPLATE_VERSION="v4.2.0"
 # https://github.com/helm/helm
 HELM_VERSION="v3.16.2"
 # https://github.com/kubernetes/kops
```

Comment:
can we bump the kops version to the latest commit 

80806a8b438fe59dbcbad9a255b890a4ac2c8dd3

https://github.com/kubernetes/kops/commit/80806a8b438fe59dbcbad9a255b890a4ac2c8dd3

Reply:
Bumped ✅